src/Cache/Engine/FileEngine.php

```diff
@@ -220,7 +220,10 @@ public function delete(string $key): bool
             return false;
         }
 
-        /** @psalm-suppress PossiblyNullReference */
+        /**
+         * @psalm-suppress PossiblyNullReference
```

Comment:
The error suppression shouldn't be necessary now that `@var` is added.

Reply:
Unfortunately, this is about the input, so another issue thats why we need to keep it.